CMakeLists.txt

```diff
@@ -180,6 +181,15 @@ if(openPMD_MPI_LINK_C)
     set(openPMD_MPI_TARGETS MPI::MPI_C ${openPMD_MPI_TARGETS})
 endif()
 
+if(openPMD_USE_CUDA STREQUAL AUTO)
```

Comment:
Let's wrap this into `if(openPMD_BUILD_EXAMPLES)`, that's the only time this is needed so far and thus will simplify build requirements.

Reply:
Changed.